### ISSUE TITLE
Allow comments in abstract code

### ISSUE THREAD

**Issue by thesamovar** [brian-team/brian2#395]
I think we don't currently allow this right? It would be good for readability I think.

**Comment by mstimberg** [comment]
Actually, I think we already do and they are included in the generated code: #261

**Comment by thesamovar** [comment]
Indeed!

They don't work on a line on their own though: perhaps that's worth adding?

e.g. this will not work:

```
reset='''
v = Vr
# oooh
'''
```

**Comment by mstimberg** [comment]
> They don't work on a line on their own though: perhaps that's worth adding?

Yes I agree that might be useful, e.g. to structure long blocks of code.

**Comment by mstimberg** [comment]
I had a quick look at this and it takes a decent amount of refactoring of the `make_statements` function (which *does* need some refactoring anyway...) to support lines without statements. Let's postpone it, I don't think it's very high priority.
